partitioner/partition.cpp

```diff
@@ -147,18 +147,14 @@ std::vector<std::vector<MeshMetaData>> partition(const MeshMetaData& mesh_meta,
                 for (uint k = 0; k < 3; ++k) {
                     if (elt_B == mesh_meta.elements.at(elt_A).neighbor_ID[k]) {
                         ElementMetaData& curr_elt = submeshes[rank_A][loc_part_A].elements.at(elt_A);
-
-                        curr_elt.neighbor_ID[k]   = DEFAULT_ID; // can we keep an actual neighbor ID?
-                        curr_elt.boundary_type[k] = Distributed<SWE::BoundaryTypes::internal>::Type();
+                        curr_elt.boundary_type[k] = Distributed(curr_elt.boundary_type[k]);
```

Comment:
@kazbekkm please double check here as well. This could be a distributed levee right?